### PR TITLE
Fix lint warnings

### PR DESCRIPTION
**Pull request by dmage**

### CODE REVIEW

File: pkg/dockerregistry/server/pullthroughblobstore.go [hunk 1]
```diff
@@ -226,15 +225,11 @@ func storeLocal(ctx context.Context, localBlobStore distribution.BlobStore, remo
...
 
-	desc, err = copyContent(remoteGetter, ctx, dgst, bw, nil)
+	desc, err = copyContent(ctx, remoteGetter, dgst, bw, nil)
 	if err != nil {
 		return err
 	}
 
 	_, err = bw.Commit(ctx, desc)
-	if err != nil {
-		return err
```
Comment (legionus) [inline pkg/dockerregistry/server/pullthroughblobstore.go]: Please Rollback/Cancel bw on error

Comment (dmage) [reply pkg/dockerregistry/server/pullthroughblobstore.go]: done

**Pull request merged**